src/effects/VST/VSTEffect.cpp

```diff
@@ -3631,7 +3634,7 @@ bool VSTEffectWrapper::StoreSettings(const VSTEffectSettings& vstSettings) const
          const auto itr = vstSettings.mParamsMap.find(pi.mName);
          if (itr != vstSettings.mParamsMap.end())
          {
-            const float& value = *itr->second;
+            const float& value = itr->second->second;
```

Comment:
The assumption that iter->second is not nullopt needs to be commented now, and the explanation is that FetchSettings passed the default true as the second argument, whenever the EffectSettings are made rather than EffectMessage.